samples/parselib/README.md

```diff
@@ -0,0 +1,51 @@
+**parselib**
+==========
+Classes for parsing and loading audio data.
+
+## Abstract
+(Oboe) ack contains facilities for reading and loading audio data from streams. Streams can be wrapped around either files or memory blocks (as from resources).
+
+**parselib** is written in C++ and is intended to be called from Android native code. It is implemented as a static library.
```

Comment:
Do you mean it is implemented using static methods or that it is built as a static library?